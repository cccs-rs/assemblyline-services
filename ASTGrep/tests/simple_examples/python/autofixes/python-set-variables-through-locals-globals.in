locals()['IlIIIllIIllIIIlllll']=aaa

globals()['sssss']=123

vars()['yes']=True
