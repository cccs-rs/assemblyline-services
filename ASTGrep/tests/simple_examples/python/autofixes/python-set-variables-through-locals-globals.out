IlIIIllIIllIIIlllll = aaa

sssss = 123

yes = True
